components/custom_status/custom_status_emoji.tsx

```diff
@@ -66,6 +70,14 @@ const CustomStatusEmoji = (props: ComponentProps) => {
                             </span>
                         }
                     </div>
+                    {customStatus.expires_at && customStatus.duration !== CustomStatusDuration.DONT_CLEAR &&
+                        <div>
+                            <span>
+                                {'Until '}
+                                {displayExpiryTime(customStatus.expires_at, timezone)}
```

Comment:
1. Maybe we should memoize this instead of evaluating this function each time?
2. I wonder if we should make it inline
```ts
const expiryText = displayExpiryTime(customStatus.expires_at, timezone);

{`Until ${expiryText}`}
```